tests/lib/collectors/grr_base.py

```diff
@@ -86,16 +86,19 @@ def testApprovalWrapper(self, _):
     # pylint: disable=protected-access,invalid-name
     grr_base_module._CHECK_APPROVAL_INTERVAL_SEC = 0
     mock_grr_object = MockGRRObject()
+    mock_telemetry = mock.MagicMock()
     mock_forbidden_function = mock.Mock(
         wraps=mock_grr_object.ForbiddenFunction)
     result = grr_base_module._WrapGRRRequestWithApproval(
-        mock_grr_object,
-        mock_forbidden_function,
-        logging.getLogger('GRRBaseModuleTest'),
-        'random1',
-        'random2',
-        random3=4,
-        random4=4)
+      mock_grr_object,
+      mock_forbidden_function,
+      logging.getLogger("GRRBaseModuleTest"),
+      mock_telemetry,
```

Comment:
Is it worth adding an assertion that telemetry gets called how you expect? (Or, possibly even adding a dedicated test for that.)